decompressed/base/etc/init.d/rootdevice

```diff
@@ -330,7 +346,7 @@ debug() {
 boot() {
   ubus send fwupgrade '{"state":"upgrading"}'
   logo_project
-  check_full_rootfs
+  check_full_overlay
```

Comment:
Dopo check_full_overlay ci starebbe bene mettersi un paio di secondi in ascolto con tcpdump su lan per qualche pacchetto trigger per forzare comunque l'erase totale

Reply:
teniamo anche questa come feature futura, altrimenti metto altra carne a cuocere e non portiamo mai a compimento questa prima cosa che è critica, il riempire rootfs è gia una cazzata generata, ma rompere l'overlay potrebbe causarlo in rari casi anche la gui